docs/usage.rst

```diff
@@ -53,14 +53,32 @@ utils.misc
 .. automodule:: tiatoolbox.utils.misc
     :members: save_yaml, split_path_name_ext, grab_files_from_dir, imwrite
 
-^^^^^^^^^^
+^^^^^^^^^^^^^^^
+utils.stainnorm
+^^^^^^^^^^^^^^^
+.. automodule:: tiatoolbox.utils.stainnorm
+    :members: ABCStainExtractor, RuifrokStainExtractor
+
+^^^^^^^^^^^^^^^^
 utils.transforms
-^^^^^^^^^^
+^^^^^^^^^^^^^^^^
 .. automodule:: tiatoolbox.utils.transforms
     :members: background_composite, imresize
 
-^^^^^^^^^^
+^^^^^^^^^^^^^^^^
 utils.exceptions
-^^^^^^^^^^
+^^^^^^^^^^^^^^^^
 .. automodule:: tiatoolbox.utils.exceptions
     :members: FileNotSupported
+
+-----
```

Comment:
Please can we move Tools before Utils? Just following alphabetical order.

Reply:
Resolved